.travis.yml

```diff
@@ -1,6 +1,15 @@
 os: linux
-dist: trusty
+dist: xenial
 language: python
+sudo: required
```

Comment:
![image](https://user-images.githubusercontent.com/18620025/84719266-e3a57900-af48-11ea-887e-8dcb2c573c7a.png)

Pretty sure the sudo line can be safely removed

Reply:
removec